src/application/clients/crux-wallet-client.ts

```diff
@@ -98,7 +98,7 @@ export const getCruxUserRepository = (options: IBlockstackCruxUserRepositoryOpti
 
 export const getPubsubClientFactory = (): IPubSubClientFactory => {
     return new CruxNetPubSubClientFactory({defaultLinkServer: {
-        host: "broker.hivemq.com",
+        host: "127.0.0.1",
```

Comment:
Don't change this in code. In code we should have a globally accessible server.

Reply:
shall we put it in config??

Reply:
We should ideally do it in the same way we've done it for Blockstack user repository. It also requires such a default for Blockstack stuff.

Look at line 127 of crux-wallet-client.ts

`this.cruxBlockstackInfrastructure = options.blockstackInfrastructure || CruxSpec.blockstack.infrastructure;`